forte/data/ontology/test/test_configs/test_additional_properties.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "test_ontology",
+    "definitions": [
+        {
+            "entry_name": "ft.onto.test_ontology.Token",
+            "parent_entry": "forte.data.ontology.top.Annotation",
+            "description": "Base parent token entry",
+            "attributes": [
+                {
+                    "name": "pos",
+                    "type": "str"
+                },
+                {
+                    "name": "pos",
+                    "type": "str"
+                }
+            ],
+            "class_attributes": [
```

Comment:
Is this test intended to show that ``class_attributes`` are not allowed?

Reply:
Yes, it is to make sure that the `additionalProperties=false` (https://json-schema.org/understanding-json-schema/reference/object.html#properties) of the JSON validation schema is working as intended.

Reply:
https://github.com/asyml/forte/pull/95/files#diff-01ba68ac9bab2f1e5d7083dedaadeaa0R42 - This test makes sure that an error is thrown if there are `class_attributes` in the schema, as the property is not allowed.